### PR TITLE
write blobs to disk if crane-version is tip

### PR DESCRIPTION
**Pull request by imjasonh**

### CODE REVIEW

File: setup-registry/action.yaml [hunk 1]
```diff
@@ -14,6 +14,11 @@ inputs:
     description: The version of crane to install.
     required: true
     default: latest-release
+  disk:
+    description: If true, use a disk-backed registry.
+    required: false
+    default: false
+
```
Comment (imjasonh) [inline setup-registry/action.yaml:21]:
```suggestion
```

File: setup-registry/action.yaml [hunk 2]
```diff
@@ -26,4 +31,4 @@ runs:
     - name: Run registry
       shell: bash
       run: |
-        PORT=${{inputs.port}} crane registry serve &
+        PORT=${{inputs.port}} crane registry serve --blobs-to-disk=${{inputs.disk}} &
```
Comment (mattmoor) [inline setup-registry/action.yaml:34]: This will only work with `tip` 😞

Comment (imjasonh) [reply setup-registry/action.yaml:33]: Documented.

Comment (mattmoor) [reply setup-registry/action.yaml:33]: My point is that even `--blobs-to-disk=false` will break without `tip`

File: setup-registry/action.yaml [hunk 3]
```diff
@@ -23,7 +27,14 @@ runs:
       uses: imjasonh/setup-crane@v0.3
       with:
         version: ${{inputs.crane-version}}
+
+    - name: Run registry
+      if: ${{inputs.disk}} == 'true' && ${{inputs.crane-version}} == 'tip'
+      shell: bash
+      run: |
+        PORT=${{inputs.port}} crane registry serve --blobs-to-disk=true &
```
Comment (mattmoor) [inline setup-registry/action.yaml:35]: So I have concerns...

Once we snap a release, how are we going to support released versions in this?

If we kill `crane-version` and instead pass `tip` to `setup-crane` when `inputs.disk` is true, then I think we have recourse to then switch to `--blobs-to-disk=${{ inputs.disk }}` after a crane release, right?

Comment (mattmoor) [reply setup-registry/action.yaml:35]: We should also add test coverage for the disk path to make sure the actions are properly predicated.

Comment (mattmoor) [reply setup-registry/action.yaml]: @imjasonh I think you did the opposite of what I was thinking 🤣

**Comment by cpanato** [comment]
@imjasonh @mattmoor, is this still needed?

**Comment by mattmoor** [comment]
no, we did something else :)

**Pull request closed without merging**